keptn-gateway/.gitignore

```diff
@@ -0,0 +1 @@
+keptn-gateway
```

Comment:
why is this here?

Reply:
convenience to ignore the go binary

Reply:
shouldn't it be under /bin then?

Reply:
I would, in case, move this under the main .gitignore file

Reply:
this ignores then the complete directory of the keptn-gateway

Reply:
you can write `./keptn-gateway/keptn-gateway`